src/context/BookingState.js

```diff
@@ -98,14 +102,16 @@ class BookingStateProvider extends React.Component<Props, BookingStateType> {
   }
 }
 
-export const withLogout = <Props>(Component: React.ComponentType<{} & Props>) =>
+export const withLogout = <Props>(
```

Comment:
this definitely doesn't belong into this file

Reply:
agreed